### PR TITLE
Organisation numbers tidying

### PR DESCRIPTION
**Pull request by stuartmccoll**
This pull request contains general tidying and slight refactoring of the controller and view for the organisation numbers page of the service.

### CODE REVIEW

File: app/views/organisation/numbers/show.html.erb [hunk 1]
```diff
@@ -1,27 +1,45 @@
...
+%>
 
-<%= render partial: "partials/summary_errors", locals: {
-        form_object: @organisation,
-        first_form_element: :organisation_company_number
-} if @organisation.errors.any? %>
+<h1 class="govuk-heading-xl" aria-label="Heading">About your organisation</h1>
 
-<%= form_with model: @organisation, url: organisation_numbers_path, method: :put do |f|  %>
+<%= form_with model: @organisation, url: :organisation_numbers, method: :put, local: true do |f|  %>
```
Comment (gidsg) [inline app/views/organisation/numbers/show.html.erb]: What is the thinking behind removing `_path` here?

File: app/views/organisation/numbers/show.html.erb [hunk 2]
```diff
@@ -31,16 +49,32 @@
...
+    <%=
+      render partial: "partials/form_input_errors",
+             locals: {
+                 form_object: @organisation,
+                 input_field_id: :charity_number
+             } if @organisation.errors[:charity_number].any?
+    %>
+
+    <%=
+      f.number_field :charity_number,
```
Comment (gidsg) [inline app/views/organisation/numbers/show.html.erb]: Should this be a text field instead now?

File: app/views/organisation/numbers/show.html.erb (hunk 1 above)
Comment (stuartmccoll) [reply app/views/organisation/numbers/show.html.erb]: The `:` means that it isn't needed - `:organisation_number` will resolve to the same thing as `organisation_numbers_path`.

File: app/views/organisation/numbers/show.html.erb (hunk 2 above)
Comment (stuartmccoll) [reply app/views/organisation/numbers/show.html.erb]: Yep, it is in `master` - this branch needs updating.

Comment (stuartmccoll) [reply app/views/organisation/numbers/show.html.erb]: Merged `master` into this branch, so this has been resolved.

**Pull request merged**